apps/etterlatte-saksbehandling-ui/client/src/components/aktivitetsplikt/brev/AktivitetspliktBrev.tsx

```diff
@@ -57,6 +65,12 @@ export function Aktivitetspliktbrev({ brevId }: { brevId: number }) {
       return (
         <GridContainer>
           <Column>
+            {endringerHarKommetEtterBrevOpprettelse(brev) && (
+              <Alert variant="warning">
+                Behandling er redigert etter brevet ble opprettet. Gå gjennom brevet og vurder om det bør tilbakestilles
```

Comment:
Vi støtter ikke tilbakestilling, så det blir litt løgn? Jeg tenker at vi kan heller si at "Vurdering av aktivitet eller valgene for infobrevet er oppdatert etter at brevet ble opprettet. Se nøye over brevet for å se om innholdet stemmer med nåværende verdier"

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/6251/commits/16a6174dddff8a25c93457ee9d22618624431338

Reply:
nice catch bruh